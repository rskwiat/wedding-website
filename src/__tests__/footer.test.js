/* eslint-disable */

import React from 'react';
import renderer from 'react-test-renderer';

import Footer from '../components/footer';

test('Testing Footer component', () => {

  const component = renderer.create(
    <Footer copyright="2018" builtBy="Testing" email="rskwiat@gmail.com" />,
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

