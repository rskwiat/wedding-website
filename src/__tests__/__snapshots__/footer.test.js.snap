// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Footer component 1`] = `
<footer>
  <div
    className="container"
  >
    <h2>
      Thank You!
    </h2>
    <h3>
      Melissa & Robert
    </h3>
    <p>
      Copyright 
      2018
       | Designed By: 
      <a
        href="mailto:rskwiat@gmail.com"
      >
        The Groom
      </a>
    </p>
  </div>
</footer>
`;
